src/components/data-exploration/differential-expression-tool/DiffExprCompute.jsx

```diff
@@ -285,14 +285,60 @@ const DiffExprCompute = (props) => {
         <Radio
           style={radioStyle}
           value={ComparisonType.WITHIN}>
-          Compare cell sets within a sample/group
+          <Space>
+            Compare cell sets within a sample/group
+            <Tooltip overlay={(
+              <span>
+                This is useful for finding differentially expressed genes when comparing cell sets (e.g. for finding marker genes that distinguish one cluster from another). The calculation uses the presto implementation of the Wilcoxon rank sum test and auROC analysis. For more information see the
+                {' '}
+                <a
+                  href='http://htmlpreview.github.io/?https://github.com/immunogenomics/presto/blob/master/docs/getting-started.html'
+                  target='_blank'
+                  rel='noreferrer'
+                >
+                  presto vignette
+                </a>.
+              </span>
+            )}
+            >
+              <InfoCircleOutlined />
+            </Tooltip>
+          </Space>
         </Radio>
         <Radio
           style={radioStyle}
           value={ComparisonType.BETWEEN}
           disabled={numSamples === 1}
         >
-          Compare a selected cell set between samples/groups
+          <Space>
+            {
+              numSamples === 1 ? (
+                <Tooltip
+                  overlay={(<span>Comparison between samples/groups is not possible with a dataset that contains only 1 sample</span>)}
```

Comment:
```suggestion
                  overlay={(<span>Comparison between samples/groups is not possible with a cluster that contains only 1 sample</span>)}
```

the dataset can have more than one sample and yet the cluster has only one sample.